java/src/test/java/com/saucelabs/simplesauce/SauceSessionTest.java

```diff
@@ -6,139 +6,148 @@
 import org.mockito.junit.MockitoJUnit;
 import org.mockito.junit.MockitoRule;
 
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.remote.BrowserType;
 import org.openqa.selenium.remote.RemoteWebDriver;
 
 import java.net.MalformedURLException;
 import java.net.URL;
 
 import static org.junit.Assert.*;
-import static org.mockito.Mockito.doReturn;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.spy;
-import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.*;
 
 public class SauceSessionTest {
-    private SauceSession sauce;
+    private SauceSession sauceSession = spy(new SauceSession());
     private RemoteWebDriver dummyRemoteDriver = mock(RemoteWebDriver.class);
-    private SauceOptions options = new SauceOptions();
+    private JavascriptExecutor dummyJSExecutor = mock(JavascriptExecutor.class);
 
     @Rule
     public MockitoRule initRule = MockitoJUnit.rule();
 
     @Before
     public void setUp() {
-        sauce = spy(new SauceSession());
-        doReturn(dummyRemoteDriver).when(sauce).createRemoteWebDriver();
+        doReturn(dummyRemoteDriver).when(sauceSession).createRemoteWebDriver();
     }
 
     @Test
-    public void sauceSession_defaultSauceOptions_returnsChromeBrowser() {
-        sauce.start();
-        String actualBrowser = sauce.getCurrentSessionCapabilities().getCapability("browserName").toString();
-        assertEquals("chrome", actualBrowser);
+    public void sauceSessionDefaultsToLatestChromeOnWindows() {
```

Comment:
Just as above, these tests should call `start()` to make sure that the object is actually created correctly

Reply:
This test checks that the correct defaults are set. 

You test that the options that were set are actually used in a separate test. What I did in Ruby was mock out the network call to test against the correct json getting sent. Figuring that out here is an option.